packages/launcher/src/widget.tsx

```diff
@@ -174,7 +174,8 @@ export class Launcher extends VDomRenderer<ILauncher.IModel> {
                 icon={icon}
                 iconClass={classes(iconClass, 'jp-Icon-cover')}
                 stylesheet="launcherSection"
-              />
+                aria-hidden="true"
```

Comment:
what is being hidden from assistive tech?

Reply:
I believe this is each of the section icons in the launcher, like so:

1. (icon: orange square containing orange bookmark) Notebook
2. (icon: blue square containing `>_`) Console
3. (icon: black square containing `$_`) Other